app/views/pages/home.html.erb

```diff
@@ -55,10 +56,26 @@
   changeBodyBackgroundColor();
 
   // Script to handle delay, switching from Splash screen to Menu screen
-  let loader = document.getElementById('splash-container');
-  const handleLoader = () => {
+
+  const handleLoader = (loader) => {
    loader.style.display = 'none';
  }
 
- setTimeout(handleLoader, 4000);
+
+
+ let handleFirstSession = () => {
```

Comment:
I'd love you to explain this to me too.  I don't think it's wrong, I just don't understand exactly what's going on here.